tests/network/mod.rs

```diff
@@ -596,21 +601,43 @@ where
     D: DistAlgorithm<Output = Batch<C, NodeId>, NodeId = NodeId>,
     C: Contribution + Clone,
 {
-    /// Verifies that all nodes' outputs agree.
+    /// Verifies that all nodes' outputs agree, noting that node 0 was removed - hence could have
+    /// missed some batches - and then rejoined the network.
```

Comment:
That's very specific to our tests. Maybe add a comment that this framework is deprecated and that's why we're not making it more general.